Kentico.Kontent.Management/Models/Shared/ListingResponseModel.cs

```diff
@@ -16,7 +16,7 @@ public class ListingResponseModel<T> : IEnumerable<T>
         private readonly string _url;
         private readonly Func<string, string, Task<IListingResponse<T>>> _nextPageRetriever;
 
-        internal ListingResponseModel(Func<string,string, Task<IListingResponse<T>>> retriever, string continuationToken, string url, IEnumerable<T> result)
+        public ListingResponseModel(Func<string, string, Task<IListingResponse<T>>> retriever, string continuationToken, string url, IEnumerable<T> result)
```

Comment:
This should not be public. Please create IListingResponseModel interface and hide this class behind it. That way it will be mockable and we won't expose any internal code.